framework/migrate/sqlite_ownerfk_test.go

```
@@ -0,0 +1,424 @@
+
+// legacyDB builds the shape a pre-v0.67 AutoMigrate produced: an entity whose
+// owner column is also a relation, so the CREATE TABLE carries a foreign key
+// the framework violates on every create.
+func legacyDB(t *testing.T, ddl ...string) *sql.DB {
+	t.Helper()
+	path := filepath.Join(t.TempDir(), "legacy.db")
+	db, err := sql.Open("sqlite3", "file:"+path)
+	if err != nil {
+		t.Fatal(err)
+	}
+	t.Cleanup(func() { db.Close() })
```

golangci-lint (2.12.2)

[error] Error return value of `db.Close` is not checked

(errcheck)